### PR TITLE
chore(IDX): bazel-bep.pb as artifact

### PR DESCRIPTION
**Pull request by marko-k0**
Upload `bazel-bep.pb` as an artifact. It will be pulled by `github-stats` for exporting bazel stats to superset.

### CODE REVIEW

File: .github/workflows-source/ci-main.yml [hunk 1]
```diff
@@ -117,6 +117,15 @@ jobs:
           if-no-files-found: error
           path: |
             bazel-targets
+      - name: Upload bazel-bep
+        uses: actions/upload-artifact@v4
+        with:
+          name: ${{ github.job }}-bep
+          retention-days: 90
+          if-no-files-found: ignore
```
Comment (nmattia) [inline .github/workflows-source/ci-main.yml:125]: in what case do we expect this file not to exist?

File: .github/workflows-source/ci-main.yml [hunk 2]
```diff
@@ -117,6 +117,15 @@ jobs:
           if-no-files-found: error
           path: |
             bazel-targets
+      - name: Upload bazel-bep
+        uses: actions/upload-artifact@v4
+        with:
+          name: ${{ github.job }}-bep
+          retention-days: 90
```
Comment (nmattia) [inline .github/workflows-source/ci-main.yml:124]: why does the retention period need to be so long?

File: .github/workflows-source/ci-main.yml [hunk 3]
```diff
@@ -117,6 +117,15 @@ jobs:
...
+          if-no-files-found: ignore
+          compression-level: 9
+          path: |
+            bazel-bep.pb
```
Comment (nmattia) [inline .github/workflows-source/ci-main.yml:128]:
```suggestion
          path: bazel-bep.pb
```

File: .github/workflows-source/ci-main.yml (hunk 1 above)
Comment (marko-k0) [reply .github/workflows-source/ci-main.yml:125]: One example would be change in README on in yaml files. See last workflow run.

File: .github/workflows-source/ci-main.yml (hunk 2 above)
Comment (marko-k0) [reply .github/workflows-source/ci-main.yml:124]: Benefit is being able to repopulate the database if we lose the data. The size of artifacts is also not big. Do you see concern here?

File: .github/workflows-source/ci-main.yml (hunk 1 above)
Comment (nmattia) [reply .github/workflows-source/ci-main.yml:125]: thanks, can you leave a note?

Comment (marko-k0) [reply .github/workflows-source/ci-main.yml:125]: Sure. Does it have to go to every job that has this step?

Comment (nmattia) [reply .github/workflows-source/ci-main.yml:125]: no no, first one should do

File: .github/workflows-source/ci-main.yml (hunk 2 above)
Comment (nmattia) [reply .github/workflows-source/ci-main.yml:124]: Yep totally fine! Though a bit unexpected; I had to double check the default value

**Review by nmattia — APPROVED** [review]

**Pull request merged**